app/src/main/res/layout/activity_main.xml

```diff
@@ -1,19 +1,244 @@
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_4"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_2"
+                    android:layout_column="@integer/column_0"
+            />
+            <Button
+                    android:text="@string/button_5"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_5"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_2"
+                    android:layout_column="@integer/column_1"
+            />
+            <Button
+                    android:text="@string/button_6"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_6"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_2"
+                    android:layout_column="@integer/column_2"
+            />
+
+            <Button
+                    android:text="@string/button_1"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_1"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_1"
+                    android:layout_column="@integer/column_0"
+            />
+            <Button
+                    android:text="@string/button_2"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_2"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_1"
+                    android:layout_column="@integer/column_1"
+            />
+            <Button
+                    android:text="@string/button_3"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_3"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_1"
+                    android:layout_column="@integer/column_2"
+            />
+
+            <Button
+                    android:text="@string/button_0"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:id="@+id/button_0"
+                    android:backgroundTint="@color/numbers"
+                    android:layout_row="@integer/row_0"
+                    android:layout_column="@integer/column_1"
+            />
+
+
```

Comment:
Unnecessary consecutive spaces. Use Android Studio's reformat code to fix them, along with the single lines with "/>"

Reply:
Done